### PR TITLE
Phrase #3724

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/phrase/sources/new-job-created/new-job-created.mjs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "New Job Created (Instant)",
+  version: "0.0.1",
+  key: "phrase-new-job-created",
+  description: "Emit new event on each job is created.",
```
Comment (andrewjschuang) [inline components/phrase/sources/new-job-created/new-job-created.mjs:8]:
```suggestion
  description: "Emit new event when each job is created.",
```

**Review by andrewjschuang — APPROVED** [review]
Thanks, LGTM!
Just some minor comments

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**